code/src/java/plugin/encounter/EncounterPlugin.java

```diff
@@ -38,15 +38,6 @@
 import javax.swing.JTabbedPane;
 import javax.swing.ListModel;
 
-import gmgen.GMGenSystem;
-import gmgen.GMGenSystemView;
-import gmgen.io.ReadXML;
-import gmgen.io.VectorTable;
-import gmgen.plugin.InitHolderList;
-import gmgen.plugin.PcgCombatant;
-import gmgen.plugin.dice.Dice;
-import gmgen.pluginmgr.messages.AddMenuItemToGMGenToolsMenuMessage;
-import gmgen.pluginmgr.messages.RequestAddTabToGMGenMessage;
```

Comment:
This is an imports ordering problem.  gmgen is part of pcgen and should be in the pcgen block.